File: CS/Form1.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace UseOneStyleController
{
	public partial class Form1
	{

		internal Form1()
		{
			InitializeComponent();
		}
		private void simpleButton1_Click(object sender, System.EventArgs e)
		{
			Form2 frm = new Form2();
			EditorsHelper.SetStyleController(frm, styleController1);
			frm.Show();
		}
	}

}

File: VB/Form1.vb
Public Class Form1

    Private Sub simpleButton1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles simpleButton1.Click
        Dim frm As New Form2
        EditorsHelper.SetStyleController(frm, styleController1)
        frm.Show()
    End Sub
End Class
